node/src/command.rs

```diff
@@ -8,7 +8,8 @@
 use crate::{
```

Comment:
This file should not be here in this PR. I will delete it. Mistakenly taken because of a local `cargo fmt`.

Reply:
let's not forger this

Reply:
removed.